package-lock.json

```diff
@@ -27871,7 +27871,7 @@
         },
         "@newfold-labs/wp-module-ecommerce": {
             "version": "git+ssh://git@github.com/newfold-labs/wp-module-ecommerce.git#a8aa51fa4fea5eb6019973149a3fbf1738e477e2",
-            "from": "@newfold-labs/wp-module-ecommerce@github:newfold-labs/wp-module-ecommerce#v0.6.4",
+            "from": "@newfold-labs/wp-module-ecommerce@github:newfold-labs/wp-module-ecommerce#v0.7.2",
```

Comment:
Can we retry `npm install` here? The git hash in the version hasn't gotten updated.